playwright/src/utilities/utilities.ts

```diff
@@ -2,19 +2,23 @@ import { ElementHandle, Page } from '@playwright/test'
 
 async function closeExtras(page: Page, retries = 5): Promise<void> {
   try {
-    await page.click('text=Got it!', { timeout: 500 })
-  } catch (error) {}
-  try {
-    await page.click('lower-sticky-note-content > .put-away', { timeout: 500 })
-  } catch (error) {}
-  try {
-    await page.click('._pi_closeButton', { timeout: 500 })
-  } catch (error) {}
-  try {
-    await page.click('.ReactModalPortal .put-away', { timeout: 500 })
+    await page.locator('text=Got it!').click()
+    // await page.click('text=Got it!', { timeout: 500 })
```

Comment:
@gregfitch Line 6 (commented) does not work but line 5 works. what is the difference

Reply:
It would probably be a context question. The reduced timeout may be ending the search for the object too soon. Functionally, they are pretty much the same except the locator enforces an element strictness that the page click does not.

a discussion on this topic:
https://github.com/microsoft/playwright/discussions/13649

and the dev doc:
https://playwright.dev/docs/locators#strictness

Reply:
yes, the reduced timeout is the issue. Are we using the timer there because we want to override the default timer value